lib/src/commands/create/create_app_command.dart

```diff
@@ -15,7 +15,7 @@ import 'package:stacked_cli/src/services/template_service.dart';
 import 'package:stacked_cli/src/templates/template_constants.dart';
 
 class CreateAppCommand extends Command {
-  final _cLog = locator<ColorizedLogService>();
+  final _log = locator<ColorizedLogService>();
```

Comment:
Lol, I wonder if it was @aymen added this _c to the log 🤣

Reply:
Probably 😄